Sprint-3/slideshow/slideshow.js

```diff
@@ -1,8 +1,37 @@
-const images = [
-    "./assets/cute-cat-a.png",
-    "./assets/cute-cat-b.jpg",
-    "./assets/cute-cat-c.jpg",
+
+/*const images = [
+  "./assets/cute-cat-1.png",
+  "./assets/cute-cat-2.jpg",
+  "./assets/cute-cat-3.jpg",
+  "./assets/cute-cat-4.jpg",
+  "./assets/cute-cat-5.jpg",
 ];
+*/
+
+// Write your code here
+
+var number_img = 1;
+var total_images = 5;
+
+function img_change_add() {
+  var img = document.getElementById("img");
+  
 
+   if (number_img < total_images) {
+     number_img += 1;
+   } else {
+     number_img = 1; 
+   }
```

Comment:
Can you figure out a way to express this in one statement (by utilising the `%` operator)?